### ISSUE TITLE
Feature request: customize sort order

### ISSUE THREAD

**Issue by KestrelWild** [Caraxi/RemindMe#26]
Would love to be able to choose the order in which skills appear in my display and to keep that order instead of resorting it based on time left on cooldown.

**Comment by meniniim** [comment]
I like this idea. However, I would prefer that while it is in cooldown, it would go to the end of the list and return to the defined priority order after becoming available.

**Comment by erikbernardo** [comment]
> I like this idea. However, I would prefer that while it is in cooldown, it would go to the end of the list and return to the defined priority order after becoming available.

that does make sense, however, is there any way we can change the "natural order" of skills?

I have a setup with basically, ogcd's and big cd's on PLD.
So currently the order I see is: Scorn, FoF, Intervene, Req, Spirits.

Is there any way that I can make FoF/Req appear before the ogcds while respecting the cds going to the end of the list?

Thanks.

**Comment by remi-gelinas** [comment]
Agreed - I would love to have the icons stay in the same order as I've set them up in, a la Weakauras :)

**Comment by DJFarr** [comment]
I'd love this one, It helps my brain work if the cooldown bars match the placement of the skills on my hotbars.

Ideally for me I'd prefer a fully fixed order, but as mentioned above, an option to move the ones on cooldown to the bottom wouldn't go astray.
